integration_testing/features/coach/coach-class-home.feature

```diff
@@ -0,0 +1,129 @@
+Feature: General navigation on class home interface
+
+  Background:
+    Given I am signed in as a class or facility coach
+      And I am on *Class home* for class <class>
+      And there are learners and coaches assigned to the class <class>
+
+  Scenario: Navigate back to the class list
+    When I press the link “All classes”
+    Then I am back on the class selection page
+      And I see a list of all the facility classes, or those I'm assigned to (if class coach)
+
+  Scenario: There are no quizzes
+    Given there are no created quizzes 
+        Then there should be a quizzes block with an empty state string saying “No quizzes assigned” # Not yet implemented?
+        And Then I should not see any quizzes
+
+  Scenario: There are no lessons
+    Given there are no created lessons
+      Then there should be a lessons block with an empty state string saying “No lessons assigned” # Not yet implemented?
+        And Then I should not see any lessons
+
+  Scenario: There has been no activity in the class
+    Given that there has not been any learner engagement in the class
+      When I look at the *Class activity* block
+      Then I see *No activity in your class* notification
+      When I click *View all* 
+      Then I am on an empty *Class activity* page
+        And I see *No activity in your class* notification
+
+  Scenario: No coaches assigned to class
+    Given there are no coaches assigned to the class
+    When I look at the <class> class home summary block at the top
+    Then I see no coaches listed
+      And I see *0* instead # Not yet implemented? Does not show anything for coaches
+
+  Scenario: No learners enrolled in the class
+    Given there are no learners assigned to the class
+      When I look at the <class> class home summary block at the top
+      Then I see *0* listed as number of learners        
+
+  Scenario: Review progress of a started quiz
+    Given there is a <quiz> quiz available
+      And there has been learner progress on a <quiz> quiz
+    When I click the <quiz> quiz progress bar # Not yet implemented?
+    Then I am on <quiz> quiz page in the *Reports* tab
+      And I see high level quiz summary data
+      And I see a list of learners with quiz progress
+
+  # Active but not started quizzes do not appear on Class home
+  Scenario: Review a not-started quiz 
```

Comment:
@rtibbles @indirectlylit This scenario was among those passed on by @khangmach, but is not implemented. Leave or remove?

Reply:
Quizzes are currently ordered on the dashboard by 'most recent activity', which seems to contradict the story here.

There are two scenarios I can think of:

* There are three or more existing quizzes with some learner activity shown in the dashboard
* There are fewer than three existing quizzes with some learner activity shown in the dashboard

In both scenarios, when a new quiz is added with no learner activity, should it appear?

Reply:
I could see the usefulness of having the active quiz that has no activity (none of the learners it is assigned to started the quiz) being listed in the **Class home** dashboard, but not seeing the necessity of reviewing it... Why would the coach want to a review quiz that no one has started yet? 
@khangmach Thoughts?

Reply:
This may have been lost in translation, but the quiz/lessons that appear in the dashboard should be **the last 3 quizzes/lessons made active by the coach user.** 

If a new quiz/lesson has just been created and made active (Boom, that's the point where it should appear both on the coach dash + learner view), naturally there will be no one starting it. However, it should still appear on the dash + learner view as a predictor of start/engagement.

Reply:
otherwise, it seems natural to me that we will get complaints about false bug positives for the most recent quiz/lesson created not showing up in the dash

Reply:
sounds reasonable, but that was not the specified behavior. I'll create a new issue to follow up

Reply:
https://github.com/learningequality/kolibri/issues/4939

question: is this important for 0.12 or should we revisit in a patch?

Reply:
I think my only concern with not using recent activity is that if a coach creates 5 lessons, and activates them all to allow students to engage at their own pace, for the first two lessons created, no activity will be shown on the dashboard.

I don't see the point of items appearing on the dashboard in anticipation - when there is activity, they will be shown.

Reply:
ref: #4937